x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_details.tsx

```diff
@@ -78,14 +91,33 @@ export const EndpointDetails = memo(
           }),
           description: details.host.os.full,
         },
+        {
+          title: i18n.translate('xpack.securitySolution.endpoint.details.agentStatus', {
+            defaultMessage: 'Agent Status',
+          }),
+          description: (
+            <EuiHealth
+              color={HOST_STATUS_TO_HEALTH_COLOR[hostStatus]}
+              data-test-subj="agentStatusHealth"
+            >
+              <EuiText size="m">
+                <FormattedMessage
+                  id="xpack.securitySolution.endpoint.list.hostStatusValue"
+                  defaultMessage="{hostStatus, select, online {Online} error {Error} unenrolling {Unenrolling} other {Offline}}"
```

Comment:
So here we use `Offline` for antyhing that is not `online`, `error` or `unenrolling`

Reply:
yup -- this is just following the pattern from the endpoint list, since it already shows Agent status. i'm not sure if we wanted to further differentiate things.